### ISSUE TITLE
Screen Reader support regression with moving users between channels

### ISSUE THREAD

**Issue by chrislgilland** [mumble-voip/mumble#6766]
### Description

When using most recent build of Mumble both on Windows 11 as well as on Ubuntu 24.10 Linux, it appears that the option when right clicking on a user to move them to another channel is no longer there. You now must use a physical mouse and drag/drop the user on to the channel destination. Example: Joe is in Root. I want to move him to Lifegroup. I must click and drag Joe, and drop him with a mouse onto the lifegroup channel. For Windows screen reader users such as those using NVDA, JAWS, Narrator,  etcetera, this is nearly impossible with the current UI. On Linux, for those using the Orca screen reader, this also is impossible at this time.

I suggest placing the move option back into the right click context menu on any user, provided the ACL promissions allow the user to carry out the task.

I can confirm this is not an ACL issue, as I checked, and further, I also cannot perform this task with SuperUser, but dragging and dropping does work. Therefore, I conclude that this is a severe accessibility regression both on Windows, and on Linux.

### Steps to reproduce

1. Open either Windows or Linux client.
2. Connect to a server.
3. Locate a user, and right click.
4. Regardless if ACL rules permit, there is no "Move" option.
Actual behavior:
User must use a physical mouse and drag the user to be moved to the new destionation.

Expected behavior:

User should be able to drag and drop, but for accessibility purpose, user also should aditionally be able to right click by means of pressing either shift+F10, or the "Applications key" to open the context right click menu, then down arrow to, and press enter on move, then be given another menu to select the channel to move the user to.

### Mumble version

1.5.634

### Mumble component

Client

### OS

Other

### Reproducible?

Yes

### Relevant log output

```shell

```

**Comment by Hartmnt** [comment]
Interesting, this has not been touched recently afaik and is definitely unintentional.

When you say "most recent build of Mumble" do you mean freshly compiled from the master branch? Or do you mean the latest pre-built release? In your ticket you mention Mumble version 1.5.634 which is the second to last stable release. Can you please double-check your client version?

**Comment by Hartmnt** [comment]
Reading this again: We have the "Move user to own channel" and "Join user's channel" context menu actions. This is what you are referring to, right?
